### ISSUE TITLE
team edition images are broken

### ISSUE THREAD

**Issue by Fodoj** [mattermost/mattermost-docker#386]
Not sure what happened here or what I am doing wrong, but current mattermost-team-edition images seem to be "broken" - if I run them, there is no `/entrypoint.sh` file and all environment variables like DB_HOST and others are just ignored and it uses connection to `dockerhost`.

Tried with 5.8, 5.8.1, 5.10.0 and latest. 

mattermost-enterprise-edition works just fine.

**Comment by pichouk** [comment]
That's right, but I cannot reproduce if I build the image from this repository. @cpanato do you know if something changed in the Docker Hub build of `mattermost-team-edition` ?

**Comment by Fodoj** [comment]
Perhaps just need to re-build and re-push?

**Comment by PFWhite** [comment]
I am seeing this same issue where the database environment variables being ignored when pulling the team image. The application gets stuck pinging localhost and eventually exits with a 102 code.

**Comment by paddatrapper** [comment]
I'm running into the same issue with 5.14.2

**Comment by paddatrapper** [comment]
[entrypoint.sh](https://github.com/mattermost/mattermost-docker/blob/master/app/entrypoint.sh#L36) uses jq to manipulate config settings before mattermost is started, but this is not installed in the docker image

**Comment by cadavre** [comment]
I just tried to migrate from `mattermost-prod-app` to `mattermost-team-edition` and ended the same.

I'll take a look at it today and try to find out what's wrong.

BTW Anyone from Mattermost Docker team – I got a question, because I can see that there's a real mess in Dockerfiles. Check this:

https://hub.docker.com/layers/mattermost/mattermost-team-edition/release-5.22/images/sha256-9088ff193b3ea31e13ff81c94a8aba3b6490cf399a43d3323fa13f5e598602a7?context=explore

vs

https://hub.docker.com/layers/mattermost/mattermost-team-edition/5.22.0/images/sha256-1ec3d6d85f25f31316a52cdfbef97c262828f6d45124cc1f07d5468932242212?context=explore

**Comment by cadavre** [comment]
@paddatrapper Docker images are now built from https://github.com/mattermost/mattermost-server/tree/master/build which entrypoint.sh is quite... empty.

**Comment by haivala** [comment]
I had same problem and I don't know if it's related but I had to chown the `volumes/app/mattermost/data` folder for the right user.
